install.sh

```diff
@@ -402,11 +402,17 @@ __install_rita() {
 __gather_OS() {
 	_OS="$(lsb_release -is)"
 	_OS_CODENAME="$(lsb_release -cs)"
+	_MONGO_OS_CODENAME="$(lsb_release -cs)"
+
 	if [ "$_OS" != "Ubuntu" -a "$_OS" != "CentOS" -a "$_OS" != "RedHatEnterpriseServer" ]; then
 		printf "$_ITEM This installer supports Ubuntu, CentOS, and RHEL. \n"
 		printf "$_IMPORTANT Your operating system is unsupported."
 		exit 1
 	fi
+
+	if [ "$_MONGO_OS_CODENAME" == "bionic" ]; then
+		_MONGO_OS_CODENAME="xenial"
```

Comment:
Could you add a comment as to why we need this? I think you said it's because we need Mongo 3.6 which isn't available in bionic but is in xenial, but otherwise reading the code it isn't obvious.

Reply:
Thanks for the feedback. Yes, I can add comments. I will do so in a moment.

Reply:
Hows that?

Reply:
I should change "keyword" to "codename", my apologies. I should change that real quick.